create_ceph_cluster_on_centos.adoc

```diff
@@ -0,0 +1,122 @@
+// vim: tw=79
+= Provisioning Ceph Cluster On CentOs
+
+This document describes the mechanism to create ceph cluster on CentOS nodes
```

Comment:
Could we have more context here? When would one need to follow this document when working with Tendrl?